### PR TITLE
Various components/shared refactoring.

### PR DESCRIPTION
**Pull request by amassarwi**
Includes:

- Ditch old `AccountsPage` redux connector.
- Add `shared/LinkToAccounts/LinkToAccounts.module.css`.
- Add `shared/UnsignedTx/UnsignedTx.module.css`.
- Replace `network` redux connector with `useNetwork` custom hook.
- Change all components' `.js` files to `.jsx`.

### CODE REVIEW

**Comment by alexlyp** [comment]
Needs a rebase

File: app/connectors/index.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
-export { default as accountsPage } from "./accountsPage";
 export { default as purchaseTickets } from "./purchaseTickets";
 export { default as send } from "./send";
 export { default as ticketsList } from "./ticketsList";
 export { default as ticketsOverview } from "./ticketsOverview";
 export { default as routing } from "./routing";
-export { default as modalVisible } from "./modalVisible";
-export { default as network } from "./network";
 export { default as lnPage } from "./lnPage";
```
Comment (alexlyp) [inline app/connectors/index.js:6]: I'm getting an error on startup.  BlurableContainer in components/layout is still referring to modalVisible.

**Review by alexlyp — CHANGES_REQUESTED** [review]

File: app/connectors/index.js (hunk 1 above)
Comment (amassarwi) [reply app/connectors/index.js:6]: Yep, rebase smoke, will let you know when all good.

Comment (amassarwi) [reply app/connectors/index.js:7]: Fixed @alexlyp

**Review by alexlyp — APPROVED** [review]

**Pull request merged**